migrations/002_migrations.py

```diff
@@ -0,0 +1,29 @@
+import peewee as pw
+from peewee_migrate import Migrator
+from contextlib import suppress
+import json
+from pathlib import Path
+from model import GameTypes
+
+with suppress(ImportError):
+    pass
+
+def migrate(migrator: Migrator, database: pw.Database, *, fake=False):
+  if Path("gametypes.json").exists():
+    try:
+        f = open("gametypes.json", encoding="utf-8")
+        gametypes = json.loads(f.read()).items()
+        f.close()
+        for gametitle, gameinfo in gametypes:
+            GameTypes.get_or_create(title=gametitle, playerCount=gameinfo["playerCount"], teamCount=gameinfo["teamCount"], statsName=gameinfo["statsName"])
+    except Exception as e:
+        print('Error: %s', repr(e)) #("No table for gametypes found! Please execute 'createdb' first.")
```

Comment:
```suggestion
        print('Error: %s', repr(e))
```
setupdb.py is not there anymore